src/api/CallAPI.tsx

```diff
@@ -0,0 +1,18 @@
+export async function CallItemSearch(page=1, pageSize=1, orderBy="favorite") {
```

Comment:
파라미터들의 타입을 지정해주는게 좋을것같아요!ㅎㅎ

```suggestion
export async function CallItemSearch(page: number, pageSize: number, orderBy: "favorite" | "someElse") {
```

Reply:
아 api부분에는 기본값을 넣어둬서 타입오류를 확인 못 해서 확인을 안해버렸네요
수정하겠습니다!